modules/get-started/pages/whats-new-241.adoc

```diff
@@ -0,0 +1,21 @@
+= What's New in 24.1 Beta
+:description: Summary of new features and updates in the release.
+
+This topic includes new content added in version 24.1 beta. 
+
+== Write caching
+
+xref:develop:config-topics.adoc#configure-write-caching[Write caching] is a relaxed mode of `acks=all` that provides better performance at the expense of durability. It acknowledges a message as soon as it is received and acknowledged on a majority of brokers, without waiting for it to fsync to disk. This provides lower latency while still ensuring that a majority of brokers acknowledge the write. For clusters in development mode, write caching is enabled by default. For clusters in production mode, it is disabled by default.
+
+== RBAC
+xref:manage:security/authorization/rbac.adoc[Role-based access control] (RBAC) provides an extension to Redpanda's existing access control lists (ACLs). It allows you to create roles and assign those roles as the authorized or denied party for an ACL. Every user you assign to that role inherits those authorizations. Using RBAC gives you a more flexible tool to manage authorization for your users, especially when you have complex organizational hierarchies, large numbers of users, or both. 
```

Comment:
'allows you to create roles, assign roles to multiple users, and assign those roles as the authorized or denied party for an ACL' 

@deniscoady @graphcareful  is this the best way to say this?

Reply:
@micheleRP would say 'flexible and efficient' (more than just 'flexible')